src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,25 @@
 import React from 'react';
 import './MoviesList.scss';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+import { MovieCard } from '../MovieCard';
+import { Movie } from '../../types/Movie';
+
+type Props = {
+  movies: Movie[];
+};
+
+export const MoviesList: React.FC<Props> = ({ movies }) => (
+  <div className="movies">
+    {movies.map(({
+      title: movieTitle, description, imgUrl, imdbUrl, imdbId,
+    }) => (
+      <MovieCard
+        title={movieTitle}
+        description={description}
+        imgUrl={imgUrl}
+        imdbUrl={imdbUrl}
+        imdbId={imdbId}
+      />
```

Comment:
Please, pay attention: you haven't used `key` for mapping, always use it when you're iterating on components